docs/content/en/docs/implementing/container.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Working with container runtime
+description: How to run tasks in container runtime
+weight: 95
+---
+
+Container runtime allows the Keptn Lifecycle Toolkit
+to execute tasks in Kubernetes containers as part of a Kubernetes Job.
+This is similar to the Keptn v1 Job Executor Service.
+
+This is a new feature introduced in v0.8.0
+that is useful for:
+
+- Running load/performance testing tools
+- Running tasks expressed in other languages such as Python
+- [TODO] What else?
+- Running SLO validations
+
+To implement this feature:
+
+- Define a
+  [KeptnTaskDefinition](../yaml-crd-ref/taskdefinition.md)
+  resource that defines the container
+- Populate a [KeptnApp](../yaml-crd-ref/app.md)
```

Comment:
```suggestion
- Populate a [KeptnApp](../yaml-crd-ref/app.md) or a workload (Pod, Deploypment, etc.)
```

Reply:
Is this true for a KeptnTaskDefinition?  I think this is just if you are creating a pure k8s container for some other purpose, right?  This particular page is gone and the info is incorporated into the implementing/tasks page; see if that information is correct.